zk/check.py

```diff
@@ -280,14 +282,13 @@ def parse_stat(self, buf):
         # body correctly. Particularly, the Connections val was added in
         # >= 3.4.4.
         start_line = buf.readline()
-        match = self.version_pattern.match(start_line)
+        match = self.version_pattern.search(start_line)
```

Comment:
given the format of the version string, using `search` instead of `match` should be just fine 👍